projects/dagster-slurm/dagster_slurm/launchers/ray.py

```diff
@@ -296,9 +338,9 @@ def _generate_cluster_template(
             echo "PAYLOAD FAILED OR SCRIPT EXITED UNEXPECTEDLY! Capturing logs..." >&2
             for node in "${{worker_nodes[@]}}"; do
                 echo "--- WORKER NODE ($node) RAYLET LOG ---" >&2
-                srun --nodes=1 --ntasks=1 -w "$node" bash -c 'tail -n 50 $(find {temp_dir_path}/session_*/logs/raylet.out -type f 2>/dev/null | sort | tail -n 1)' || echo "Worker log on $node not found." >&2
+                srun --nodes=1 --ntasks=1 --cpu-bind=none -w "$node" bash -lc 'tail -n 50 $(find {temp_dir_path}/session_*/logs/raylet.out -type f 2>/dev/null | sort | tail -n 1)' || echo "Worker log on $node not found." >&2
```

Comment:
why are you overwriting the changes we have from https://github.com/ascii-supply-networks/dagster-slurm/pull/60 which already enable configurable cpu binds?